web/src/tippyjs.js

```diff
@@ -414,9 +423,7 @@ export function initialize() {
         appendTo: () => document.body,
     });
 
-    delegate("body", {
-        target: ".message_inline_image > a > img",
-        appendTo: () => document.body,
+    message_list_tooltip(".message_inline_image > a > img", {
         // Add a short delay so the user can mouseover several inline images without
```

Comment:
We can also remove mutation observer from here.

Reply:
Sure, I removed it. I wasn't sure if the mutation observer was only for rerendering or if it was for other cases where we want this particular tooltip to hide. Because of this, I left this particular observer untouched and removed all the others. However, I have now removed it.